contrib/aws/aws-sdk-go-v2/aws/aws.go

```diff
@@ -101,6 +123,324 @@ func (mw *traceMiddleware) startTraceMiddleware(stack *middleware.Stack) error {
 	}), middleware.After)
 }
 
+func extractResourceNameFromParams(requestInput middleware.InitializeInput, awsService string) (resourceNameKey string, resourceNameValue string) {
+	switch awsService {
+	case "SQS":
+		resourceNameKey, resourceNameValue = extractQueueName(requestInput)
```

Comment:
nitpik:
Personally, I would prefer writing it this way:
```go
resourceNameKey = tagQueueName
resourceNameValue = extractQueueName(requestInput)
```
because logically `resourceNameKey = tagQueueName` is determined by this switch case here instead of the `extractQueueName` method. The same applies to all other methods.

Reply:
yeah that makes sense, added a new commit :)